### DISCUSSION TITLE
23.02  Azure and SAML integration

### DISCUSSION THREAD

**Discussion by mfenton01** [opendcim/openDCIM#1424] [Q&A] [answered]
Concerning just the Authentication . It is not mentioned anywhere that I can find .  Do should the Apache Basic Authentication be disabled as well ? 

One other item when the SAML Success page refreshes it never actually makes it to the dashboard it just keeps repeating success page. To  break out of it I have to use the base url .   Https://dcim.yourdomain.com

Not sure if this is related to above question but regardless of the userid  type used  ( ID or email )  the account still gets created which creates a duplicate . It seems Azure  sending the principal name or email ,  DCIM doesn't recognize it and makes new.  I can go in after the fact by changing the db.inc back to basic authentication and update the rights .  When I change it back to SAML I then get the results expected.

**Comment by samilliken** [comment, marked as answer]
You should only have one type of authentication enabled.

 

From: mfenton01 
Sent: Monday, August 21, 2023 10:51 AM
To: opendcim/openDCIM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [opendcim/openDCIM] 23.02 Azure and SAML integration (Discussion #1424)

 

Concerning just the Authentication . It is not mentioned anywhere that I can find . Do should the Apache Basic Authentication be disabled as well ?

One other item when the SAML Success page refreshes it never actually makes it to the dashboard it just keeps repeating success page. To break out of it I have to use the base url . Https://dcim.yourdomain.com

Not sure if this is related to above question but regardless of the userid type used ( ID or email ) the account still gets created which creates a duplicate . It seems Azure sending the principal name or email , DCIM doesn't recognize it and makes new. I can go in after the fact by changing the db.inc back to basic authentication and update the rights . When I change it back to SAML I then get the results expected.

**Comment by mfenton01** [comment]
I do .. I only have just SAML I commented in the db.inc

I am referring to the Apache  SSL  configuration . conf file  right now it
has auth type is set to basic..

**Reply by samilliken** [reply]
So.   Ask yourself.   Would you consider that another form of authentication?

**Comment by mfenton01** [comment]
Ok  :-)   if you put it like that, yes .
